src/components/error-boundary/error-boundary.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+export class ErrorBoundary extends React.Component {
+    constructor(props) {
+      super(props);
+      this.state = { hasError: false };
+    }
+  
+    static getDerivedStateFromError(error) {
+      // Update state so the next render will show the fallback UI.
```

Comment:
Let's avoid usage of such comments since they are not really useful during work. Good code is a code that can describe itself with clear names and logic